django_socio_grpc/settings.py

```diff
@@ -110,7 +112,7 @@ class FilterAndPaginationBehaviorOptions(str, Enum):
     # Variable that indicate the class used to generate the name of the proto messages
     "DEFAULT_MESSAGE_NAME_CONSTRUCTOR": "django_socio_grpc.protobuf.message_name_constructor.DefaultMessageNameConstructor",
     # Variable that indicate the plugins used in proto generation by default
-    "DEFAULT_GENERATION_PLUGINS": [],
+    "DEFAULT_GENERATION_PLUGINS": [GlobalScopeWrappedEnumGenerationPlugin()],
```

Comment:
to discuss with @legau but I think the default should be enum in global scope not wrapped. If possible to detect a duplicate and print a warning it's event better.

Reply:
@legau reminder to answer this conv

Reply:
DEFAULT_GENERATION_PLUGINS are going to get overriden by the `use_generation_plugins` grpcaction attribute, we might need a BASE_GENERATION_PLUGINS that are always there

Reply:
Doing this will effectively prevent us from overriding the Enum generation plugin in grpcaction, and i don't think we want that.
the other way around would be to add a parameter in grpcaction :
- `use_generation_plugins `-> override the `DEFAULT_GENERATION_PLUGINS`
- `extend_generation_plugins `-> add new plugins to the `DEFAULT_GENERATION_PLUGINS`

Reply:
I agree with @Giiltham 

Also the base comment on this thread was more about the defaut enum plugin more than the plugin override but both subject need to be discussed

Reply:
i'll give my 2 cents on the subject, if i have two enums containing a single member that is common , like a YES or NO, it would not work well without wrapped, and i don't think it's a safe default way to handle enums in the .proto. 
It should be pretty normal to have dupplicated enums member throughout an application ?

Reply:
instead of a new "extend_generation_plugins" parameter we will add a boolean "override_generation_plugins" False by default so that it doesn't introduce breaking changes for us, and still has the same usage. @AMontagu